src/linter/block.go

```diff
@@ -438,9 +438,7 @@ func (b *BlockWalker) EnterNode(w walker.Walkable) (res bool) {
 			res = false
 		}
 	case *stmt.Interface:
-		if b.ignoreFunctionBodies {
-			res = false
-		}
+		res = b.handleInterface(s)
```

Comment:
`handleInterface` always returns `false`.
I believe it should return `!b.ignoreFunctionBodies`.

Otherwise, the block walker will check function bodies during the top-level traversal.